shared/src/store/action-reducers/simulation.ts

```diff
@@ -0,0 +1,136 @@
+import { IsNumber, IsUUID, Min } from 'class-validator';
+import type {
+    TreatPatientsBehaviorState,
+    UnloadArrivingVehiclesBehaviorState,
+} from '../../simulation';
+import type { Mutable } from '../../utils';
+import { UUID, uuidValidationOptions } from '../../utils';
+import { IsValue } from '../../utils/validators';
+import type { Action, ActionReducer } from '../action-reducer';
+import { ReducerError } from '../reducer-error';
+import { getElement } from './utils';
+
+export class UpdateTreatPatientsIntervalsAction implements Action {
```

Comment:
Remove this action and it's reducer since it will be added (slightly different) by #720.